__tests__/Unit/Functions/fetchIdleUsers.test.ts

```diff
@@ -0,0 +1,34 @@
+import { IDLE_USERS_URL } from '@/components/constants/url';
+import fetchIdleUsers from '@/helperFunctions/fetchIdleUsers';
+
+describe('fetchIdleUsers', () => {
+  it('should return an array of idle user names and error flag when the API call is successful', async () => {
+    const expectedIdleUserNames = ['tom', 'jerry'];
+    const data = {
```

Comment:
Please use msw to mock the data and not mock fetch. Refer https://kentcdodds.com/blog/stop-mocking-fetch

The PR that fixes our msw bug is https://github.com/Real-Dev-Squad/website-status/pull/435